pkgs/thycotic/dsv-cli/registry.yaml

```diff
@@ -0,0 +1,31 @@
+packages:
+  - type: github_release
+    repo_owner: thycotic
+    repo_name: dsv-cli
+    asset: dsv-{{.OS}}-{{.Arch}}
+    format: raw
+    replacements:
+      amd64: x64
+      arm64: x64
+      386: x86
+      arm64n: arm64
+      windows: win
+    supported_envs:
+      - darwin
+      - linux
+      - windows
```

Comment:
```suggestion
      - amd64
```

`linux/arm64` and `windows/arm64` aren't supported,
so you shouldn't add `linux` and `windows`.

`amd64` includes `linux/amd64` and `windows/amd64`, so it is enough.